portal-api/src/v1/users/validation/rules/select-at-least-one-role.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Validates that if the roles are present at least one is not null
+ * @param {Object} user the existing user
+ * @param {Object} change the changes to make
+ * @returns {Array} either an empty array or an array containing an error object
+ */
+const selectAtLeastOneRole = (user, change) => {
+  if (
+    !change.roles ||
```

Comment:
Please simplify this condition, this is overly complicate.

Reply:
The condition is simplified.